frontend_lib/i18next.scanner/fr/translation.json

```diff
@@ -98,7 +98,9 @@
   "This word is reserved for group mentions": "Ce mot est réservé aux mentions de groupe",
   "Error while checking username availability": "Erreur lors de la vérification de la disponibilité de l'identifiant",
   "This username is not available": "Cet identifiant n'est pas disponible",
-  "You are trying to mention an invalid user": "Vous essayez de mentionner un utilisateur non valide",
   "Are you sure?": "Êtes-vous sûr ?",
-  "all": "tous"
+  "Confirm": "Confirmer",
+  "pending": "en attente",
+  "rejected": "rejetée",
```

Comment:
For subscription requests we use "réfusée". Should we keep the same terminology?

Reply:
In fact we already have both:
- {{author}} vous a refusé l'accès à {{space}}
- {{author}} a rejeté l'accès à {{space}} pour {{user}}
I'll edit them to only keep "refusé".

Reply:
Done.